### PR TITLE
Caprover integration

### PR DESCRIPTION
**Pull request by finnus**

### CODE REVIEW

**Comment by finnus** [comment]
squash commits would be nice here, as there are so many trial and error commits

**Comment by finnus** [comment]
I finally added an extra dockerfile for caprover. so that the old setup and docker-compose procedure should still work

File: compose/production/django/entrypoint [hunk 1]
```diff
@@ -6,37 +6,17 @@ set -o nounset
...
+export CELERY_BROKER_URL="${REDIS_URL}"
+
 
 if [ -z "${POSTGRES_USER}" ]; then
     base_postgres_image_default_user='postgres'
     export POSTGRES_USER="${base_postgres_image_default_user}"
 fi
 export DATABASE_URL="postgis://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}"
 
-postgres_ready() {
```
Comment (henhuy) [inline compose/production/django/entrypoint:16]: Why did you remove postgres ready check?

**Review by henhuy — APPROVED** [review]
Did you adapted new Dockerfile-caprover file from current cookiecutter Dockerfile?
The old one was my special Dockerfile building up from conda and poetry - I think this is outdated. So your solution is more up to date.
Nevertheless, I think having two Dockerfiles for production is contra-productive as only one will be used and the other probably is getting old/neglected and will become legacy.
Maybe best option is to remove original Dockerfile completely?
The rest is fine for me (I did not tested to set it up myself!).

File: compose/production/django/entrypoint (hunk 1 above)
Comment (finnus) [reply compose/production/django/entrypoint:16]: Thanks for pointing that out. I must have disabled it at some point. But it should be back. (see next commit).

**Comment by finnus** [comment]
I wanted to keep the old deploy mechanism intact, in case you ever want to do deploy it the old way (without caprover).
The dockerimage there is really only for caprover. It doesn't come from cookiecutter, as they dont use poetry, which was some pain to resolve.
The caprover-dockerimage isn't really smart and quite big, but it works.
